### PR TITLE
Update gradle build system to a working state.

### PR DESCRIPTION
**Pull request by showlabor**
Hi,

this is my first pull request so I don't know if I'm doing it right ;-)

I've updated the gradle build system for pd-for-android so that it is actually working. Currently you can build PdCore, CircleOfFifths, ScenePlayer and Voice-O-Rama. I don't know if there's something missing.

I'll send a corresponding pull request for btmidi.

Regards,

Felix Homann

### CODE REVIEW

File: CircleOfFifths/build.gradle [hunk 1]
```diff
@@ -3,21 +3,22 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:0.4'
+        classpath 'com.android.tools.build:gradle:0.6+'
```
Comment (residuum) [inline CircleOfFifths/build.gradle]: Android studio version >=1.0.0 needs gradle at least at 0.6+

File: PdCore/build.gradle [hunk 2]
```diff
@@ -31,5 +34,24 @@ android {
         }
 
         instrumentTest.setRoot('tests')
-    }
+    }	
+}
+
+dependencies {
+    compile project(':midi:AndroidMidi')
```
Comment (residuum) [inline PdCore/build.gradle]: This line is not working for me, but is working with

    compile project(':AndroidMidi')

May be related to gradle version 1.0.0 (see other comment).

**Comment by residuum** [comment]
You should also not check in the binary file gradle/wrapper/gradle-wrapper.jar

Otherwise, the patch is working for me (TM).

**Comment by joebowbeer** [comment]
Check out pull request #10 to make sure it contains all the best stuff: gradle-plugin is at rev. 1.2.3, etc.

According to the [guide](http://gradle.org/docs/current/userguide/gradle_wrapper.html), <code>gradle-wrapper.jar</code> __should__ be checked in, right?

I think it is important that <code>build.gradle</code> handles the ndk-build part as well. Unfortunately, the current ndk support in the Android gradle plugin is deprecated, and the recommended practice (at the moment) is to call ndk-build directly in a custom Exec task (which I have done).

Going forward, a PdCore .aar should be available from a maven repo, which will really simplify things for consumers of this library.

**Pull request closed without merging**